src/main/openapi/ds-discover-openapi_v1.yaml

```diff
@@ -558,6 +558,99 @@ paths:
                   Output compatible with [Solr CSV response](https://solr.apache.org/guide/8_10/response-writers.html#csv-response-writer)
                 type: string
 
+
+  /solr/{collection}/suggest:
+    get:
+      tags:
+        - '${project.name}'
+      summary: 'Perform a Solr suggest in the stated collection and dictionary'
+      operationId: solrSuggest
+      parameters:
+
+        - name: collection
+          in: path
+          description: 'The ID of the Solr collection to search. Available collections can be requested from /solr/admin/collections'
+          required: true
+          schema:
+            type: string
+            default: 'ds'
+            example: 'ds'
+
+
+        - name: suggestDictionary
+          in: query
+          description: 'The suggest dictionary for use. The dictionary must be defined in solr.'
+          required: true
+          schema:
+            type: string
+            default: 'dr_title_suggest'
+            example: 'dr_title_suggest'
+
+        - name: suggestQuery
+          in: query
+          description: 'The query text for suggest. Minimum 2 characters must be present. No solr operators allowed'
+          required: true
+          schema:
+            type: string
+            default: ''
+          examples:
+            lowercase:
+              value: 'tv'
+              summary: 'All documents starting with tv (case sensivive)'
+            uppercase:
+              value: 'TV'
+              summary: 'All documents starting with TV (case sensivive)'
+             
+        - name: suggest.count
+          in: query
+          description: |             
+            The number suggestions to return. The default is 10
+          required: false
+          schema:
+            type: integer
+            format: int32
+            example: 10
+            default: 10
+
+        - name: wt
+          in: query
+          description: | 
+            Solr response writer [wt](https://solr.apache.org/guide/8_10/response-writers.html)
+
+            Controls whether the response delivery format. Common values (non exhaustive list) are
```

Comment:
"whether" should be removed. This error exists 2 other places in the OpenAPI YAML, so please also correct it there.

Reply:
fixed, also other methods.